sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -399,6 +399,8 @@ Returns an `annotated_ref` reference wrapper which can be used to read or write
 to the underlying pointer. Reads/Writes using the reference will retain the
 annotations.
 
+Only applicable properties will be passed to `annotated_ref`.  Unless a property's description states otherwise it will not be passed on.  
+
```

Comment:
Hi Lueck. Since the annotated_ptr currently serve two purpose right now:
1. interface annotation
2. pointer annotation

Here we tried to split properties for interface annotation and pointer annotation as well. We want all annotations to be carried by annotated_ptr, but only passing pointer annotations to annotated_ref

Does this make sense to you?

Reply:
This doesn't make sense to me. Are you talking about alignment or some other property?
For alignment whether we filter has nothing to do with ref vs ptr or interface vs pointer but whether it has an offset.

Reply:
No I am not talking about alignment here. These are regarding these two types
1. interface annotation:  awidth, dwidth, buffer_location
2. pointer annotation: buffer_location, alignment

All properties are carried by annotated_ptr, while we only passing pointer annotation to annotated_ref

Reply:
I think either filtering out the property in header file or in SPIRV translator works. But filtering out these properties in header file will be easier and more maintainable.